witness/src/Dscp/Snowdrop/ReadMode.hs

```diff
@@ -0,0 +1,53 @@
+-- | SD extended reading mode.
+
+module Dscp.Snowdrop.ReadMode
```

Comment:
Why do we call it "mode", anyway?

Reply:
It provides two modes for read-only actions: chain-only and chain+mempool. Do you have alternatives?